docs/socketexample.rst

```diff
@@ -552,7 +569,7 @@ need this method.
 .. doctest::
 
    >>> utils = list(zope.component.getAllUtilitiesRegisteredFor(IUSSocket))
-   >>> utils.sort()
+   >>> utils.sort(key=lambda x: type(x).__name__)
```

Comment:
You could bind the result of `zope.component.getAllUtilitiesRegisterFor(IUSSocket)` to a temporary, e.g.:

```python
>>> all_utils = zope.component.getAllUtilitiesRegisteredFor(IUSSocket)
>>> utils = sorted(all_utils, key=lambda x: type(x).__name__)
```